### ISSUE TITLE
Config File uses the wrong template

### ISSUE THREAD

**Issue by REclipsent** [NikitaCartes/Glowing-Torchflower#7]
The config file that is generated in the place of the one for this mod seems to be the one that is supposed to be for another mod by the same creator if it's installed first on that instance. 
```
##                          ##
##    Glowing Eyeblossom    ##
##    Main Configuration    ##
##                          ##

# Brightness of the open eyeblossom
# Default: 12
open-eyeblossom-brightness: ${openEyeblossomBrightness}

# Brightness of the closed eyeblossom
# Default: 3
closed-eyeblossom-brightness: ${closedEyeblossomBrightness}

# Brightness of the open eyeblossom in flowerpot
# Default: 14
open-eyeblossom-flowerpot-brightness: ${openEyeblossomFlowerpotBrightness}

# Brightness of the closed eyeblossom in flowerpot
# Default: 3
closed-eyeblossom-flowerpot-brightness: ${closedEyeblossomFlowerpotBrightness}

# Config Version. Used for automatic migration of config files.
# Do not change this value manually.
config-version: 1
```

This can cause a crash with Mod Menu installed along side it with this stack trace:
```
java.lang.ExceptionInInitializerError
	at knot//net.minecraft.class_2358.method_10199(class_2358.java:336)
	at knot//net.minecraft.class_2966.method_12851(class_2966.java:55)
	at knot//net.minecraft.client.main.Main.main(Main.java:139)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.RuntimeException: [GlowingTorchflower] Failed to load config file
	at knot//xyz.nikitacartes.glowingtorchflower.config.ConfigTemplate.loadConfig(ConfigTemplate.java:33)
	at knot//xyz.nikitacartes.glowingtorchflower.config.MainConfigV1.load(MainConfigV1.java:27)
	at knot//net.minecraft.class_2246.modifyExpressionValue$bfa000$glowing-torchflower$modifyTorchflower(class_2246.java:2401)
	at knot//net.minecraft.class_2246.<clinit>(class_2246.java:315)
	... 9 more
Caused by: org.spongepowered.configurate.loader.ParsingException: []@(line 3, col 0): null
Reader: 3
^
	at knot//org.spongepowered.configurate.hocon.HoconConfigurationLoader.loadInternal(HoconConfigurationLoader.java:175)
	at knot//org.spongepowered.configurate.hocon.HoconConfigurationLoader.loadInternal(HoconConfigurationLoader.java:61)
	at knot//org.spongepowered.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:155)
	at knot//org.spongepowered.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:63)
	at knot//org.spongepowered.configurate.loader.ConfigurationLoader.load(ConfigurationLoader.java:56)
	at knot//xyz.nikitacartes.glowingtorchflower.config.ConfigTemplate.loadConfig(ConfigTemplate.java:31)
	... 12 more
Caused by: com.typesafe.config.ConfigException$UnresolvedSubstitution: Reader: 3: Could not resolve substitution to a value: ${openEyeblossomBrightness}
	at knot//com.typesafe.config.impl.ConfigReference.resolveSubstitutions(ConfigReference.java:111)
	at knot//com.typesafe.config.impl.ResolveContext.realResolve(ResolveContext.java:183)
	at knot//com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:146)
	at knot//com.typesafe.config.impl.SimpleConfigObject$ResolveModifier.modifyChildMayThrow(SimpleConfigObject.java:380)
	at knot//com.typesafe.config.impl.SimpleConfigObject.modifyMayThrow(SimpleConfigObject.java:313)
	at knot//com.typesafe.config.impl.SimpleConfigObject.resolveSubstitutions(SimpleConfigObject.java:399)
	at knot//com.typesafe.config.impl.ResolveContext.realResolve(ResolveContext.java:183)
	at knot//com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:146)
	at knot//com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:235)
	at knot//com.typesafe.config.impl.SimpleConfig.resolveWith(SimpleConfig.java:79)
	at knot//com.typesafe.config.impl.SimpleConfig.resolve(SimpleConfig.java:69)
	at knot//com.typesafe.config.impl.SimpleConfig.resolve(SimpleConfig.java:64)
	at knot//com.typesafe.config.impl.SimpleConfig.resolve(SimpleConfig.java:42)
	at knot//org.spongepowered.configurate.hocon.HoconConfigurationLoader.loadInternal(HoconConfigurationLoader.java:173)
	... 17 more
```

**Comment by NikitaCartes** [comment]
I will fix it now, but does this happen every time? Because I can't reproduce it

**Comment by NikitaCartes** [comment]
Fixed with 1.3.0, thanks for creating this issue

**Comment by REclipsent** [comment]
I was using the Prism Launcher so I don't know if that effected how the config files were being made. But I was able to get it constantly by:
- Installing Eyeblossom
- Launching the game
- Installing Torchflower
- Relaunching
- This would result in the messed up config file

But all seems to be good now, thanks
